### ISSUE TITLE
Reproducibility on Airport (node classification) 

### ISSUE THREAD

**Issue by andyweizhao** [HazyResearch/hgcn#42]
Hi, I am running HGCN on Airport (with the command below) in the node classification task. Sadly, I received ~77.86 accuracy on test set, far below ~90.6 reported in the paper. 

"python train.py --task nc --dataset airport --model HGCN --lr 0.01 --dim 16 --num-layers 2 --act relu --bias 1 --dropout 0.0 --weight-decay 0 --manifold PoincareBall --log-freq 5 --cuda 0 --c None" -- adapted from the Airport command in README provided for link prediction.

I believe I made mistakes. Any hints are very appreciated!

**Comment by robertjankowski** [comment]
This is quite a late comment. I also ran HGCN on Airport, and the accuracy I got was 78.05, which is far below the reported one.  

Should we use different hyperparameters for this dataset? Or pretrained it like Cora?
